src/display/editor/bezier_approx.js

```diff
@@ -0,0 +1,677 @@
+/**
```

Comment:
First of all, that's *a lot* of code needed to support adding of Ink Annotations.

Secondly, it looks like this is available through NPM so can we please use that instead of "dumping" this code into the PDF.js repository (since that's not very maintainable)?

Reply:
> First of all, that's _a lot_ of code needed to support adding of Ink Annotations.
> 
Yes it is but it helps to make the drawing nicer and it helps to reduce the size of the final `InkList` when the annotation is saved.

Reply:
Do you have an example on how to use a node package in pdf.js ?

Reply:
Just including a Node.js package in a *build* is trivial, but that unfortunately doesn't take the `gulp server` mode into account. However that should be possible to solve, but it's possibly non-trivial to implement; note e.g. [the following development-specific build tasks](https://github.com/mozilla/pdf.js/blob/135b9fbcfb2e2554754082d482341b4cbe9e1b43/gulpfile.js#L1921-L1966).

In order to be able to move forward with this PR a bit more quickly, I'd accept that we *temporarily* add this code to the PDF.js library assuming the following:
 - The new file is placed in its own directory in https://github.com/mozilla/pdf.js/tree/master/external/, and *not* inside of the `src/`-folder.
 - The automated comment in https://github.com/mozilla/pdf.js/pull/14989#pullrequestreview-995957102 is fixed.
 - An issue is opened, which tracks removing that and using the npm package instead.